package.json

```diff
@@ -63,14 +63,14 @@
     "formik": "^2.2.6",
     "gh-pages": "^2.2.0",
     "microbundle-crl": "^0.13.10",
-    "node-sass": "^4.0.0",
+    "sass": "^1.44.0",
```

Comment:
doesnt look like sass is actually used in this. I think it was in the peerDeps just because of Pebble